src/styles/settings/_colors.scss

```diff
@@ -0,0 +1,6 @@
+$colors: (
```

Comment:
Nice work with the map. If possible I would try and find a more descriptive name for the colors. Ideally it would have something to do with what they're used for, but its not a big deal.